src/js/render.js

```diff
@@ -0,0 +1,81 @@
+/**
+	Pullquote
+*/ 
+
+
+// Knight Lab Namespace
+KL = {};
+
+// Debug Mode
+KL.debug = true;
+
+/*	KL.Bind
+================================================== */
+KL.Bind = function (/*Function*/ fn, /*Object*/ obj) /*-> Object*/ {
+	return function () {
+		return fn.apply(obj, arguments);
+	};
+};
+
+/*	Required Files
+	Webpack
+	https://webpack.github.io/
+================================================== */
+
+// CORE
+KL.Util = 		require("core/KL.Util.js");
+KL.Class = 		require("core/KL.Class");
+
+// DOM
+KL.DomMixins = 	require("dom/KL.DomMixins");
+KL.Dom = 		require("dom/KL.Dom");
+KL.DomUtil = 	require("dom/KL.DomUtil");
+KL.DomEvent = 	require("dom/KL.DomEvent");
+
+KL.PullquoteRender = (function() {
+
+	// DOM ELEMENTS
+	this.el = {
+		composition: 			KL.Dom.get("kl-quote-comp"),
+		quote_text: 			KL.Dom.get("kl-quote-text"),
+		cite: 					KL.Dom.get("kl-quote-cite"),
+		image: 					KL.Dom.get("kl-quote-image")
+	};
+
+	// DATA
```

Comment:
Feels repetitive, we're including this in render.js. Is there some other way we can define these defaults globally?